src/utils/helpers/assignment/getPreviousStageAssignment.ts

```diff
@@ -0,0 +1,15 @@
+import { ReviewAssignmentStatus } from '../../generated/graphql'
+import { AssignmentDetails } from '../../types'
+
+export const getPreviousStageAssignment = (
+  assignments: AssignmentDetails[],
+  currentStageNumber?: number
+) => {
+  const previousStage = (currentStageNumber || 1) - 1
+  return assignments
+    .filter((assignment) => assignment.current.stage.number === previousStage)
+    .filter(
+      ({ isLastLevel, current: { assignmentStatus } }) =>
+        assignmentStatus === ReviewAssignmentStatus.Assigned && isLastLevel
+    )[0] // TODO: Deal with case of more than one lastLevel in stage
```

Comment:
> // TODO: Deal with case of more than one lastLevel in stage

Do you mean more than one lastLevel review assignment? (Maybe make comment a little clearer)

Reply:
Improved adding a log when the problem described happens - so we know something odd has happened